### ISSUE TITLE
-inf or Zero loss values

### ISSUE THREAD

**Issue by Mojzaar** [gtegner/mine-pytorch#3]
Hi,

Thank you for providing your paper's code. 
I'm quite curious about how I will use MINE in my code. 
When I use MINE torch model to train the generator and add the output to my Wasserstein loss, I get either -inf (if I don't use gradient normalization) or zero, indicating that the generator output and the real image are perfectly matched (am I right?) 
Do you have any ideas about why this is occurring?

**Comment by anonymous102410** [comment]
Hi,
Did you solve your problem?
Could you tell me how to solve it?
thanks
